code/modules/projectiles/boxes_magazines/external/pistol.dm

```diff
@@ -25,6 +25,13 @@
 	desc= "A gun magazine. Loaded with rounds which penetrate armour, but are less effective against normal targets."
 	ammo_type = /obj/item/ammo_casing/c10mm/ap
 
+/obj/item/ammo_box/magazine/m10mm/makeshift
+	name = "makeshift pistol magazine (10mm)"
+	desc = "A hastily made gun magazine that can only store 4 bullets."
+	icon_state = "9x19pM"
+	max_ammo = 4
+	start_empty = TRUE
```

Comment:
Not needed

Reply:
Even if I make the makeshift pistol single-shot, a makeshift magazine would be handy.  You can sometimes find guns with no magazine and it's difficult to render them usable.